packages/turf-mask/test.js

```diff
@@ -0,0 +1,56 @@
+var mask = require('./');
+var test = require('tape');
+var poly = {
+    'type': 'Feature',
+    'properties': {},
+    'geometry': {
+        'type': 'Polygon',
+        'coordinates': [[[100, 0], [101, 0], [101, 1], [100, 1], [100, 0]]]
+    }
+};
+
+var maskPoly = {
+    'type': 'Feature',
+    'properties': {},
+    'geometry': {
+        'type': 'Polygon',
+        'coordinates': [[100.2, 0.2], [100.8, 0.2], [100.8, 0.8], [100.2, 0.8], [100.2, 0.2]]
+    }
+};
+
+
+var maskedOutputWorld = {
+    'type': 'Feature',
+    'properties': {},
+    'geometry': {
+        'type': 'Polygon',
+        'coordinates': [
+          [[100, 0], [101, 0], [101, 1], [100, 1], [100, 0]],
+          [[180, 90], [-180, 90], [-180, -90], [180, -90], [180, 90]]
+        ]
+    }
+};
+
+var maskedOutputSpecific = {
+    'type': 'Feature',
+    'properties': {},
+    'geometry': {
+        'type': 'Polygon',
+        'coordinates': [
+          [[100, 0], [101, 0], [101, 1], [100, 1], [100, 0]],
+          [[100.2, 0.2], [100.8, 0.2], [100.8, 0.8], [100.2, 0.8], [100.2, 0.2]]
+        ]
+    }
+};
+
+test('turf collect module', function (t) {
+    var maskedWithNoMask = mask(poly);
+    t.deepEqual(maskedWithNoMask, maskedOutputWorld, 'Masks to the world');
```

Comment:
Try to make your tests dynamic, hardcoding Polygons in the test cases is very difficult to troubleshoot and you can't easily expand the test cases by simply adding another folder with a couple geojsons.

I've converted a few modules in TurfJS so far with this concept.

You can also check out the latest one [`turf-line-intersect`](https://github.com/Turfjs/turf/blob/master/packages/turf-line-intersect/test.js).

Everything that's an INPUT should be placed in `test/in` and the `test/out` should be autogenerated by adding the environment variable **REGEN=true**.